source/_posts/2016-12-01-jq-the-cool-tool-for-school.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "jq - The Cool Tool For School"
+date: 2016-12-01 08:00:17 -0700
+comments: true
+author: "Wen Li <wen.li@grandrounds.com>"
+categories: tooling, cli
+---
+There’s something about using a good tool for the need. If you deal with JSON and you haven’t played with jq , you’re in for a real treat.
```

Comment:
extra space between "jq" and comma. first sentence is a little awkward, maybe try something like, "There's something incredibly satisfying about finding the perfect tool for a job."